pydcs_extensions/uh60l/uh60l.py

```diff
@@ -0,0 +1,1292 @@
+            US_Army_MEDEVAC = "US Army MEDEVAC"
+            US_Army_SOAR = "US Army SOAR"
+            US_Navy_Grey = "US Navy Grey"
+            USAF_Rescue = "USAF Rescue"
+            US_Coast_Guard = "US Coast Guard"
+            X51 = "X51"
+
+        class NorthKorea(Enum):
+            default = "default"
+            FAB = "FAB"
+            Israeli_Air_Force = "Israeli Air Force"
+            JASDF_SAR_Yellow = "JASDF SAR Yellow"
+            Skyward = "Skyward"
+            US_Army_Desert = "US Army Desert"
+            US_Army_MEDEVAC = "US Army MEDEVAC"
+            US_Army_SOAR = "US Army SOAR"
+            US_Navy_Grey = "US Navy Grey"
+            USAF_Rescue = "USAF Rescue"
+            US_Coast_Guard = "US Coast Guard"
+            X51 = "X51"
+
+        class Ethiopia(Enum):
+            default = "default"
+            FAB = "FAB"
+            Israeli_Air_Force = "Israeli Air Force"
+            JASDF_SAR_Yellow = "JASDF SAR Yellow"
+            Skyward = "Skyward"
+            US_Army_Desert = "US Army Desert"
+            US_Army_MEDEVAC = "US Army MEDEVAC"
+            US_Army_SOAR = "US Army SOAR"
+            US_Navy_Grey = "US Navy Grey"
+            USAF_Rescue = "USAF Rescue"
+            US_Coast_Guard = "US Coast Guard"
+            X51 = "X51"
+
+        class Japan(Enum):
+            default = "default"
+            FAB = "FAB"
+            Israeli_Air_Force = "Israeli Air Force"
+            JASDF_SAR_Yellow = "JASDF SAR Yellow"
+            Skyward = "Skyward"
+            US_Army_Desert = "US Army Desert"
+            US_Army_MEDEVAC = "US Army MEDEVAC"
+            US_Army_SOAR = "US Army SOAR"
+            US_Navy_Grey = "US Navy Grey"
+            USAF_Rescue = "USAF Rescue"
+            US_Coast_Guard = "US Coast Guard"
+            X51 = "X51"
+
+        class Thailand(Enum):
+            default = "default"
+            FAB = "FAB"
+            Israeli_Air_Force = "Israeli Air Force"
+            JASDF_SAR_Yellow = "JASDF SAR Yellow"
+            Skyward = "Skyward"
+            US_Army_Desert = "US Army Desert"
+            US_Army_MEDEVAC = "US Army MEDEVAC"
+            US_Army_SOAR = "US Army SOAR"
+            US_Navy_Grey = "US Navy Grey"
+            USAF_Rescue = "USAF Rescue"
+            US_Coast_Guard = "US Coast Guard"
+            X51 = "X51"
+
+    class Pylon1:
+        CEFS_Fuel_Tank_200_gallons = (1, WeaponsUH60L.CEFS_Fuel_Tank_200_gallons)
+
+    # ERRR <CLEAN>
+
+    class Pylon2:
+        CEFS_Fuel_Tank_200_gallons = (2, WeaponsUH60L.CEFS_Fuel_Tank_200_gallons)
+
+    # ERRR <CLEAN>
+
+    class Pylon3:
+        CEFS_Fuel_Tank_200_gallons = (3, WeaponsUH60L.CEFS_Fuel_Tank_200_gallons)
+
+    # ERRR <CLEAN>
+
+    class Pylon4:
+        CEFS_Fuel_Tank_200_gallons = (4, WeaponsUH60L.CEFS_Fuel_Tank_200_gallons)
+
+    # ERRR <CLEAN>
+
+    pylons = {1, 2, 3, 4}
+
+    tasks = [task.Transport, task.Reconnaissance]
+    task_default = task.Transport
+
+
+class KC130J(PlaneType):
```

Comment:
Really? The UH-60 mod also adds a second KC-130?

Reply:
Yeah, it's because DCS apparently can't task the normal tankers to fly slow enough to comfortably tank the helo. You _can_ tank from the vanilla KC-130, but it's apparently hard (as if AAR in a helicopter wasn't hard enough).

AFAIK, it's just a copy of the normal KC-130 but with tweaked speed properties.